plugins/sync/src/main/kotlin/app/aaps/plugins/sync/garmin/LoopHub.kt

```diff
@@ -29,6 +35,9 @@ interface LoopHub {
     /** Returns the factor by which the basal rate is currently raised (> 1) or lowered (< 1). */
     val temporaryBasal: Double
 
+    /** Returns the temporary basal rate in percent */
+    val temporaryBasalPercent: String
```

Comment:
I think this is UI logic so would put it closer to the UI, i.e. on the device in this case. Why is this a string?
Anyway, leave it if you need it but please add a test.